airgun/browser.py

```diff
@@ -670,7 +670,13 @@ def move_to_element(self, locator, *args, **kwargs):
         el = self.element(locator, *args, **kwargs)
         self.execute_script(
             "arguments[0].scrollIntoView(false);", el, silent=True)
-        ActionChains(self.selenium).move_to_element(el).perform()
+        try:
+            ActionChains(self.selenium).move_to_element(el).perform()
+        except selenium.common.exceptions.JavascriptException as e:
+            if 'javascript error: Cannot read property \'left\' of undefined' in e.msg:
+                pass
```

Comment:
Can you add following line?
```python
self.logger.debug("Ignoring 'Cannot read property left of undefined' exception")
```

Ignoring for a while that we fail to capture these logs, I believe we want to monitor them, so eventually we might be able to fix the root cause. We should put required pieces already, even if we end up removing the method.

Reply:
sure, that's a good idea